obstacles_x : [-9,9,5, -1, 1, 2, -7 , 6,-2, 2, -8, 0, 9]
obstacles_y : [0,9,6,-5 ,1, 7,4, 2, 4, -3,8,0, 9]
starting_pose : [5, -5]
goal_pose : [-10, 10]
